server/routers/userRouter.js

```diff
@@ -11,6 +11,13 @@ userRouter.route("/get/").get(async (req, res) => {
   res.json(user);
 });
 
+userRouter.route("/get-dietitian/").get(async (req, res) => {
+  const dietitian = await userModel.find({
+    isdietitian: true,
+  });
+  res.json(dietitian);
```

Comment:
use return statement with status code such as: return res.status(200).send(dietitian) for the succession of try and some corresponding status 400 for the catch

Reply:
Used status code